src/main/java/spoon/reflect/visitor/filter/AbstractFilter.java

```diff
@@ -31,6 +31,7 @@
 	/**
 	 * Creates a filter with the type of the potentially matching elements.
 	 */
+	// TODO: INFER TYPE BY INTROSPECTION
```

Comment:
I just copied that, but I would also like to have to simpler. Could somebody add a comment which will show how the calling code will look like after it is fixed?